src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,10 +158,14 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
 the units are assumed to be mm.
 
+### Volume
+
 | Coordinate System                                   | Description                                                                                                                                                                                                                                                                                                                                                      |
-| ---------------------------------------------------   | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   |
+| --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FS305                                               | FreeSurfer variant of the MNI305 space                                                                                                                                                                                                                                                                                                                           |
 | MNI152Lin                                           | Also known as ICBM (version with linear coregistration) [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152Lin](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152Lin)                                                                                                                                                                                       |
-| MNI152NLin6\[Sym&#124;Asym\]                        | Also known as ICBM 6th generation (non-linear coregistration). Used by SPM99 -   SPM8 and FSL (MNI152NLin6Sym). [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6)                                                                                                                           |
+| MNI152NLin6\[Sym&#124;Asym\]                        | Also known as ICBM 6th generation (non-linear coregistration). Used by SPM99 - SPM8 and FSL (MNI152NLin6Sym). [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6)                                                                                                                             |
```

Comment:
This is incorrect

```suggestion
| MNI152NLin6Sym                                      | Also known as ICBM152 6th generation (non-linear coregistration). [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6)                                                                                                                             |
| FSLMNI152NLin6Asym                                  | A variation of the ``MNI152NLin6Sym`` template that is distributed and used by FSL.                                                                                                                                                  |
```

Refs.
https://github.com/templateflow/templateflow/issues/6#issuecomment-454648955
https://github.com/templateflow/templateflow/issues/6#issuecomment-455000224

The SPM bit might also be wrong. I'm not as positive here, but I think SPM has a custom derivation of the MNINLin6Sym template. Paging @gllmflndn here.

Reply:
I don't think we can replace `MNI152NLin6Asym`  with `FSLMNI152NLin6Asym` due to backward compatibility, but we can add it if you want.

Reply:
Ooooh. I was meaning to ask you guys about this! I got confused last week when the templateflow templates for fmriprep didn't work with my HCP pipeline steps (they use FSL)!

Reply:
> I don't think we can replace `MNI152NLin6Asym`  with `FSLMNI152NLin6Asym` due to backward compatibility, but we can add it if you want.

I see, yes. I think a deprecation warning here would also be nice. I think it is crucial to call for attention on this detail.